### DISCUSSION TITLE
error：No module named 'ttkbootstrap.constants'

### DISCUSSION THREAD

**Discussion by chineselzh** [israel-dryer/ttkbootstrap#227] [Q&A]
when i run the code:
```python
import tkinter as tk
import ttkbootstrap as ttk
from ttkbootstrap.constants import *

root = tk.Tk()

b1 = ttk.Button(root, text="Button 1", bootstyle=SUCCESS)
b1.pack(side=LEFT, padx=5, pady=10)

b2 = ttk.Button(root, text="Button 2", bootstyle=(INFO, OUTLINE))
b2.pack(side=LEFT, padx=5, pady=10)

root.mainloop()
```
I get the error：`No module named 'ttkbootstrap.constants`
how can i use ttkbootstrap.constants?

![image](https://user-images.githubusercontent.com/11897604/158151269-428c5608-d85c-4d5a-901d-3a18f1b4f044.png)

**Comment by israel-dryer** [comment]
From your error message it appears that ttkbootstrap is not installed.

https://ttkbootstrap.readthedocs.io/en/latest/gettingstarted/installation/

**Reply by marioshaya** [reply]
Me too, I got this problem

I have it installed.

**Comment by enharyong** [comment]
change your script filename to anything other than ttkbootstrap.py LOL

**Reply by jongden** [reply]
> change your script filename to anything other than ttkbootstrap.py LOL
That's the reason. Thank you

**Comment by chribonn** [comment]
In my case the problem was that the virtual environment (.venv) already existed. I deleted it and recreated within VS code.

**Comment by hishamdalal** [comment]
install ttkbootstrap globally:

this will not work:
`(env) d:\my_project pip install ttkbootstrap 
`
this will work:
`d:\my_project pip install ttkbootstrap
`

**Comment by TheDemonGuard** [comment]
@chineselzh 

I cannot reproduce the issue with the sample script provided by you.
Can you retry with the latest ttkbootstrap version.
<img width="205" height="104" alt="image" src="https://github.com/user-attachments/assets/9934e668-bbbf-4153-8a2d-dddf6fa59fea" />

Additionally, it seems that the python script name you created is "ttkbootstrap.py",
you need to rename your script name other than "ttkbootstrap".
Because I think this is confusing the import statement.

> when i run the code:
> 
> ```python
> import tkinter as tk
> import ttkbootstrap as ttk
> from ttkbootstrap.constants import *
> 
> root = tk.Tk()
> 
> b1 = ttk.Button(root, text="Button 1", bootstyle=SUCCESS)
> b1.pack(side=LEFT, padx=5, pady=10)
> 
> b2 = ttk.Button(root, text="Button 2", bootstyle=(INFO, OUTLINE))
> b2.pack(side=LEFT, padx=5, pady=10)
> 
> root.mainloop()
> ```
> 
> I get the error：`No module named 'ttkbootstrap.constants` how can i use ttkbootstrap.constants?
> 
> ![image](https://user-images.githubusercontent.com/11897604/158151269-428c5608-d85c-4d5a-901d-3a18f1b4f044.png)

**Reply by TheDemonGuard** [reply]
Additionally, to check whether `ttkbootstrap` is installed correctly, please execute the following command.

`python -m ttkbootstrap`

This should display a window something like the following.

<img width="974" height="807" alt="image" src="https://github.com/user-attachments/assets/dd5366a3-c396-4276-81c3-a81d61c6f106" />
